### PR TITLE
Adding some text tweaks to publicly visible text in the samples.

### PR DESCRIPTION
**Pull request by egilmorez**
Adding some text tweaks to publicly visible text in the samples. Thanks for your review!

### CODE REVIEW

**Review by leoortizz — APPROVED** [review]

File: starters/angular/basic/src/app/pages/deferrable-views/deferrable-views.component.html [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-  Deferrable views can be used in component template to defer the loading of
+  Deferrable views can be used in a component template to defer the loading of
   select dependencies within that template. Those dependencies include
   components, directives, and pipes, and any associated CSS. To use this
   feature, you can declaratively wrap a section of your template in a
   <code>&#64;defer</code> block which specifies the loading conditions.
   Deferrable views support a series of triggers, prefetching, and several sub
   blocks used for placeholder, loading, and error state management. You can also
-  create custom conditions with when and prefetch when.
+  create custom conditions with <code>when</code> and <code>prefetch when</code>.
```
Comment (egilmorez) [inline starters/angular/basic/src/app/pages/deferrable-views/deferrable-views.component.html:11]: Is this correct -- those are literals?

I probably have this wrong, but it needs some revision as it stands.  Help!  :)

Comment (leoortizz) [reply starters/angular/basic/src/app/pages/deferrable-views/deferrable-views.component.html:11]: cc @9kubczas4

Comment (9kubczas4) [reply starters/angular/basic/src/app/pages/deferrable-views/deferrable-views.component.html:11]: Exactly, the proposed change is correct

**Review by 9kubczas4 — APPROVED** [review]
LGTM

File: starters/angular/basic/src/app/pages/deferrable-views/deferrable-views.component.html (hunk 1 above)
Comment (egilmorez) [reply starters/angular/basic/src/app/pages/deferrable-views/deferrable-views.component.html:11]: Thanks!

**Comment by H9-tate** [comment]
can u fix hosting

**Pull request merged**